src/contexts/accountContext.tsx

```diff
@@ -67,6 +67,7 @@ interface AccountAuthenticatedResponseJson
   fname: string;
   email: string;
   birthday?: string;
+  agePref?: number[] | null;
```

Comment:
can we remove the `null` type here. I think that would be best

Reply:
Yes it actually broke the build so I just pushed that fix